Classes/Controllers/PBGitCommitController.m

```diff
@@ -112,19 +115,25 @@ - (IBAction)signOff:(id)sender
 {
 	NSError *error = nil;
 	GTConfiguration *config = [repository.gtRepo configurationWithError:&error];
-	NSString* userName = [config stringForKey:@"user.name"];
-	NSString* userEmail = [config stringForKey:@"user.email"];
-	if (!(userName && userEmail))
-		return [[repository windowController] showMessageSheet:@"User's name not set" infoText:@"Signing off a commit requires setting user.name and user.email in your git config"];
-	NSString *SOBline = [NSString stringWithFormat:@"Signed-off-by: %@ <%@>",
-				userName,
-				userEmail];
+	NSString *userName = [config stringForKey:@"user.name"];
+	NSString *userEmail = [config stringForKey:@"user.email"];
+	if (!(userName && userEmail)) {
+		return [[repository windowController]
+				showMessageSheet:NSLocalizedString(@"User‘s name not set",
+												   @"Title for sheet that the user’s name is not set in the git configuration")
```

Comment:
And the tabstop width can be set [with ?ts=4](https://github.com/gitx/gitx/pull/30/files/746a76e4c4e21e3a600fa0f4bb0426e5e04e2c56..d043eaeddfe1f22778d312ee29e4aee1037df022?ts=4).

Xcode *almost* does the nice method of tabs for indentation, spaces for alignment. Instead it does tabs for indentation, both tabs and spaces for alignment...

![vim](https://cloud.githubusercontent.com/assets/34699/23044227/e89bd708-f464-11e6-8961-4b8ec5160368.png)

(vim with `list`/`listchars`)